demos/supabase/supabase/seed.sql

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Minor, but technically these should be in a Supabase migration file and the seed file would be used for injecting a user/test data. But this gets the job done.

Reply:
That does align better with Supabase - this is now implemented